monitor.go

```diff
@@ -25,10 +25,13 @@ var dockerUsername = flag.String("username", "", "Registry username for pulling
 var dockerPassword = flag.String("password", "", "Registry password for pulling and pushing")
 var registryHost = flag.String("registry-host", "", "Hostname of the registry being monitored")
 var repository = flag.String("repository", "", "Repository on the registry to pull and push")
+var baseImage = flag.String("base-image", "", "Docker image to use as base image when preparing image for push")
```

Comment:
Please make it clear this is optional and `base-layer-id` can be used instead (and vice versa)